mmpose/models/detectors/top_down_crowd.py

```diff
@@ -0,0 +1,346 @@
+import math
+
+import cv2
+import mmcv
+import numpy as np
+import torch
+from mmcv.image import imwrite
+from mmcv.visualization.image import imshow
+
+from mmpose.core.evaluation import pose_pck_accuracy
+from mmpose.core.evaluation.top_down_eval import crowd_keypoints_from_heatmaps
+from mmpose.core.post_processing import flip_back
+from .. import builder
+from ..registry import POSENETS
+from .base import BasePose
+
+
+@POSENETS.register_module()
+class TopDownCrowd(BasePose):
+    """Top-down crowd pose detectors.
+
+    Args:
+        backbone (dict): Backbone modules to extract feature.
+        keypoint_head (dict): Keypoint head to process feature.
+        train_cfg (dict): Config for training. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path to the pretrained models.
+        loss_pose (dict): Config for loss. Default: None.
+    """
+
+    def __init__(self,
+                 backbone,
+                 keypoint_head=None,
+                 train_cfg=None,
+                 test_cfg=None,
+                 pretrained=None,
+                 loss_pose=None):
+        super().__init__()
+
+        self.backbone = builder.build_backbone(backbone)
+
+        if keypoint_head is not None:
+            self.keypoint_head = builder.build_head(keypoint_head)
+        self.train_cfg = train_cfg
+        self.test_cfg = test_cfg
+
+        self.loss = builder.build_loss(loss_pose)
+        self.init_weights(pretrained=pretrained)
+
+    @property
+    def with_keypoint(self):
```

Comment:
avoid starting with `with`. `with` makes user think of the python with-statement.

maybe the name `has_keypoint` or `use_keypoint` or some other possible `xxx_keypoint` is better

Reply:
a second thought. with_xxx is not that bad.